script/run-cypress-tests-gha.js

```diff
@@ -0,0 +1,19 @@
+const { runCommand } = require('./utils');
+const exec = require('child_process').exec;
+
+exec("find src -name '*.cypress.*.js' | tr '\n' ','", function(_err, stdout) {
+  const strings = stdout.split(',').sort();
```

Comment:
This part could be done with `fs.readdirSync` if you want to avoid string manipulation and read the list of files directly in Node as an array.